pkg/webhook/server.go

```diff
@@ -54,9 +54,10 @@ type ServerOptions struct {
 	// Client will be injected by the manager if not set.
 	Client client.Client
 
-	// InstallWebhookConfig controls if the server will automatically create webhook related objects
+	// DisableInstallWebhookConfig controls if the server will automatically create webhook related objects
 	// during bootstrapping. e.g. webhookConfiguration, service and secret.
-	InstallWebhookConfig bool
+	// If false, the server will install the webhook config objects. It is defaulted to false.
+	DisableInstallWebhookConfig *bool
```

Comment:
Something about the name feels off while reading it. "Disable" and "Install" not going together. One option could be "DisableWebhookConfigAutoInstall" ?